katas/content/multi_qubit_gates/fredkin_gate/solution.md

```diff
@@ -0,0 +1,23 @@
+Again this is essentially bookwork, because there is only one gate that performs this state change (and the task title already gave it away!)  
```

Comment:
I think the other bookwork task is after this one (they're swapped here compared to the BasicGates kata), so let's rephrase
```suggestion
This is essentially bookwork, because there is only one gate that performs this state change (and the task title already gave it away!)  
```